src/components/pages/home/slider/MainSlider.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable max-len */
+import { bannersData } from "../../../../mocks/featured-banners-data";
+import { useEffect, useState } from "react";
+
+//Components
+import { BtnSlider } from "../../../common/BtnSlider";
+import { Slider } from "../../../common/Slide";
+
+const banners = bannersData.results;
+
+export function MainSlider() {
+  const [slideIndex, setSlideIndex] = useState(1);
+  const [paused, setPaused] = useState(false);
+
+  const nextSlide = () => {
+    if (slideIndex !== banners.length) {
+      setSlideIndex(slideIndex + 1);
+    } else if (slideIndex === banners.length) {
+      setSlideIndex(1);
+    }
+  };
```

Comment:
When the condition is `true` the function `setSlideIndex` is called, and when the condition is `false` the function `setSlideIndex` is also called, what's changing in here is just the parameters with which the function is called, this can be handled using the ternary.

Additionally, the conditions used in the `if\else` statement are mutually exclusive, i. e. if one happens the other one cannot occur, they cannot happen at the same time. This makes the second condition unnecessary

The logic can be changed to:

```suggestion
  const nextSlide = () => {
    const newIndex = slideIndex !== banners.length
      ? slideIndex + 1
      : 1;
    setSlideIndex(newIndex);
  };
```

Reply:
Solved